src/Interpreters/DDLWorker.cpp

```diff
@@ -535,6 +535,7 @@ bool DDLWorker::tryExecuteQuery(DDLTaskBase & task, const ZooKeeperPtr & zookeep
                                  e.code() != ErrorCodes::TABLE_IS_READ_ONLY &&
                                  e.code() != ErrorCodes::CANNOT_ASSIGN_ALTER &&
                                  e.code() != ErrorCodes::CANNOT_ALLOCATE_MEMORY &&
+                                 e.code() != ErrorCodes::TOO_MANY_SIMULTANEOUS_QUERIES &&
```

Comment:
```
2025-02-19 00:35:59 /repo/src/Interpreters/DDLWorker.cpp:538:58: error: no member named 'TOO_MANY_SIMULTANEOUS_QUERIES' in namespace 'DB::ErrorCodes'
2025-02-19 00:35:59   538 |                                  e.code() != ErrorCodes::TOO_MANY_SIMULTANEOUS_QUERIES &&
```

You need to add this as `extern` above

Reply:
I'll take a look at that. Is this a situation where these are more client-type errors ?

Reply:
I added it and seems it is improved the build, but other issues remain